src/PKSim.Core/Model/IndividualProteinTask.cs

```diff
@@ -0,0 +1,273 @@
+               relExpParam(REL_EXP),
+               initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_LUMEN));
+         }
+      }
+
+      protected virtual MoleculeExpressionContainer AddContainerExpressionNew(IContainer parentContainer, string containerName, string moleculeName,
+         string groupingName,
+         params ParameterMetaData[] parameters)
+      {
+         return addContainerExpressionNew(parentContainer, containerName, moleculeName, groupingName, parameters);
+      }
+
+      protected IParameter createConstantParameterInNew(IContainer parameterContainer, 
+         ParameterValueMetaData parameterValueDefinition,
+         string groupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
+         bool canBeVaried = true,
+         bool canBeVariedInPopulation = true,
+         bool visible = true,
+         string displayUnit = null)
+         
+      {
+
+         var parameter = _parameterFactory.CreateFor(parameterValueDefinition);
+         parameter.GroupName = groupName;
+         parameter.CanBeVaried = canBeVaried;
+         parameter.CanBeVariedInPopulation = canBeVariedInPopulation;
+         parameter.Visible = visible;
+         parameterContainer.Add(parameter);
+         if (displayUnit != null)
+            parameter.DisplayUnit = parameter.Dimension.Unit(displayUnit);
+
+         return parameter;
+      }
+
+      private MoleculeExpressionContainer addContainerExpressionNew(IContainer parentContainer, string containerName, string moleculeName,
+         string groupingName,
+         IEnumerable<ParameterMetaData> parameters)
+      {
+         var expressionContainer = createContainerExpressionForNew(parentContainer, moleculeName);
+         expressionContainer.GroupName = groupingName;
+         expressionContainer.ContainerName = containerName;
+         parameters.Each(p => addParameterIn(expressionContainer, p, moleculeName));
+         return expressionContainer;
+      }
+
+      protected void addParameterIn(IContainer container, ParameterMetaData parameterMetaData, string moleculeName, string groupName = null)
+      {
+         switch (parameterMetaData)
+         {
+            case ParameterRateMetaData rateMetaData:
+               createFormulaParameterIn(container, rateMetaData, moleculeName, groupName);
+               break;
+            case ParameterValueMetaData parameterValueMetaData:
+               createConstantParameterInNew(container, parameterValueMetaData, groupName);
+               break;
+         }
+      }
+
+      protected void AddVascularSystemExpressionNew(IContainer moleculeContainer, string groupName, params ParameterMetaData[] parameters)
+      {
+         parameters.Each(p => addParameterIn(moleculeContainer, p, moleculeContainer.Name, groupName));
+      }
+
+      private MoleculeExpressionContainer createContainerExpressionForNew(IContainer parentContainer, string containerName)
```

Comment:
There is still a mix of old and new here. Once I am done it will be consistent

Reply:
ah, ok  -that's why this xxxNEW-naming?